subsys/net/lib/mqtt_sn/mqtt_sn_transport_udp.c

```diff
@@ -25,11 +25,13 @@ static char *get_ip_str(const struct sockaddr *sa, char *s, size_t maxlen)
 {
 	switch (sa->sa_family) {
 	case AF_INET:
-		inet_ntop(AF_INET, &(((struct sockaddr_in *)sa)->sin_addr), s, maxlen);
+		zsock_inet_ntop(AF_INET, &(((struct sockaddr_in *)sa)->sin_addr),
```

Comment:
Maybe a stupid question and I'm on mobile, but if this removes a dependency, shouldn't a header include or a config be removed as well?

Reply:
True, there was one redundant header include left in the file. We can remove POSIX requirement from the sample as well, will update.